cloud/aws/templates/aws_oidc/bin/awsSetupTemplate.py

```diff
@@ -0,0 +1,46 @@
+import os
+import subprocess
+import shlex
+import shutil
+
+from cloud.shared.bin.lib.setup_template import SetupTemplate
+from cloud.aws.bin.lib import backend_setup
+
+
+class AwsSetupTemplate(SetupTemplate):
+
+    def _tf_run_for_aws(self, is_destroy=False):
+        template_dir = os.path.join(self.config.get_template_dir(), 'setup')
+        print(" - Copy the tfvars file into the setup dir")
+        shutil.copy2(
+            os.path.join(
+                self.config.get_template_dir(), self.config.tfvars_filename),
+            os.path.join(
+                self.config.get_template_dir(), 'setup',
+                self.config.tfvars_filename))
+        print(" - Run terraform init")
+        subprocess.check_call(
```

Comment:
since you're doing this... should we use shared/bin/lib/terraform.py everywhere?